### PR TITLE
Updated framework and refactor service worker file

### PR DESCRIPTION
**Pull request by Deokumarjnu**
Ticket - quintype/ace-planning#512

### CODE REVIEW

**Review by ags1773 — APPROVED** [review]

File: app/isomorphic/app-version.js [hunk 1]
```diff
@@ -1 +1 @@
-module.exports = 1;
+module.exports = 2;
```
Comment (Athira001) [inline app/isomorphic/app-version.js:1]: This is for testing right?

File: views/js/service-worker.ejs [hunk 2]
```diff
@@ -5,16 +5,17 @@ const REQUIRED_ASSETS = [
...
   // Put fonts here
-  {url: '/shell.html', revision: '<%= assetHash("app.js") %>'}
+  "/shell.html?revision=<%= assetHash("app.js") %>-<%= configVersion %>"
 ];
 
-const workbox = initializeQServiceWorker(self, {
+initializeQServiceWorker({
   routes: <%- JSON.stringify(routes) %>,
   assets: REQUIRED_ASSETS,
+  shell: "/shell.html?revision=<%= assetHash("app.js") %>-<%= configVersion %>",
```
Comment (Athira001) [inline views/js/service-worker.ejs:14]: port to a variable and resuse

File: app/isomorphic/app-version.js (hunk 1 above)
Comment (Deokumarjnu) [reply app/isomorphic/app-version.js:1]: yes. you ask me to update this, not sure what you wanted to test. will revert it.

Comment (Athira001) [reply app/isomorphic/app-version.js:1]: if appVersion got change service worker should update.

File: views/js/service-worker.ejs (hunk 2 above)
Comment (Deokumarjnu) [reply views/js/service-worker.ejs:14]: what is reusability here?

File: views/js/service-worker.ejs [hunk 3]
```diff
@@ -1,20 +1,23 @@
 <%- serviceWorkerHelper %>;
 
+cosnt shellUrl = "/shell.html?revision=<%= assetHash("app.js") %>-<%= configVersion %>";
```
Comment (Athira001) [inline views/js/service-worker.ejs:3]: typo

**Review by Athira001 — APPROVED** [review]

**Pull request merged**